xc7/litex_demo/arty.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
```

Comment:
why do we need this file here? Can't we use target from litex-boards? We clone it (and specify the exact version of it). Is there any reason to copy this file here?

Reply:
The main reason was that I couldn't find another way to use another arty board (in this case arty-100), so that we can have both the 35t and 100t tested

Reply:
push the `--board-variant` upstream. This looks like a feature we should there (and remove the arty.py file from this PR)

Reply:
Sure, PR opened here: https://github.com/enjoy-digital/litex/pull/750

Reply:
Done. Script removed and litex updated to have the board variant.